libshaderc/include/shaderc/env.h

```diff
@@ -44,9 +43,6 @@ typedef enum {
   // See glslang/Standalone/Standalone.cpp
   // TODO(dneto): Glslang doesn't accept a OpenGL client version of 460.
   shaderc_env_version_opengl_4_5 = 450,
-  // Currently WebGPU doesn't have versioning, since it isn't finalized. This
-  // will have to be updated once the spec is finished.
-  shaderc_env_version_webgpu,
```

Comment:
Same here.  Please retain the enum but cause uses of it to fail.

Reply:
Done